src/stdlib_experimental_stats_mean.fypp

```diff
@@ -19,7 +19,7 @@ contains
         ${t1}$ :: res
 
         if (.not.optval(mask, .true.)) then
-          res = ieee_value(real(res, kind=${k1}$), ieee_quiet_nan)
+          res = ieee_value(1._${k1}$, ieee_quiet_nan)
```

Comment:
Note: this line (as written before) generated a Run-Time Check Failure with `ifort` because `res` is used without being defined.